StandAlone/main.cpp

```diff
@@ -74,9 +74,9 @@ int main(int argc, char *argv[]){
         auto camera =xpcfComponentManager->resolve<input::devices::ICamera>();
         auto binaryMarker =xpcfComponentManager->resolve<input::files::IMarker2DSquaredBinary>();
 
-        auto imageViewer =xpcfComponentManager->resolve<display::IImageViewer>();
-        auto imageViewerGrey =xpcfComponentManager->resolve<display::IImageViewer>();
-        auto imageViewerBinary =xpcfComponentManager->resolve<display::IImageViewer>();
+        auto imageViewer =xpcfComponentManager->resolve<display::IImageViewer>("original");
+        auto imageViewerGrey =xpcfComponentManager->resolve<display::IImageViewer>("binary");
+        auto imageViewerBinary =xpcfComponentManager->resolve<display::IImageViewer>("grey");
```

Comment:
There is a confusion about the name
```suggestion
        auto imageViewerGrey =xpcfComponentManager->resolve<display::IImageViewer>("grey");
        auto imageViewerBinary =xpcfComponentManager->resolve<display::IImageViewer>("binary");
```